draft-ietf-quic-transport.md

```diff
@@ -851,6 +851,12 @@ limits are set in the transport parameters (see
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
 unidirectional and bidirectional streams.
 
+If a max_streams transport parameter or MAX_STREAMS frame is received with a
+value greater than 2^60, this allows a maximum stream ID that cannot be
+expressed as a variable-length integer (see {{integer-encoding}}).
+If either is received, the connection MUST be immediately closed with a
```

Comment:
"be immediately closed" feels like odd word order, and doesn't occur elsewhere in the document.  Maybe "be closed immediately" or simply "be closed"?

Reply:
Changed to closed immediately